src/main/java/seedu/address/MainApp.java

```diff
@@ -112,8 +118,11 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ReadOnlyDiary initialDiary;
         initialDiary = initDiary(storage);
 
+        ReadOnlyDashboard initialDashboard;
+        initialDashboard = initDashboard(storage);
+
         return new ModelManager(initialDukeCooks, initialHealthRecords, initialRecipeBook, initialWorkoutPlanner,
-                initialDiary, userPrefs);
+                initialDiary, userPrefs, initialDashboard);
```

Comment:
I think for consistency, you should put the initialDashboard field before userPrefs in the ModelManager constructor